tsdb/tombstones/tombstones.go

```diff
@@ -317,12 +317,10 @@ func (itvs Intervals) Add(n Interval) Intervals {
 		}
 
 		if n.Mint < r.Mint {
-			newRange := make(Intervals, i, len(itvs[:i])+1)
-			copy(newRange, itvs[:i])
-			newRange = append(newRange, n)
-			newRange = append(newRange, itvs[i:]...)
-
-			return newRange
+			itvs = append(itvs, Interval{})
```

Comment:
This is changing the backing store of the itvs slice, rather than creating a new one. Is this safe?

Reply:
Is it the same with `itvs = append(itvs, n)`?  I am not sure about this.

Reply:
True, we're altering it already.